src/pages/Home/Home.jsx

```diff
@@ -7,15 +7,15 @@ function Home() {
     <div className="home">
       <div className="home__content">
         <h1 className="home__title">Who are we?</h1>
-        <p className="home__description">
+        <span className="home__description">
```

Comment:
Maybe you are switching to `<span>` because there cannot be an `<hr>` inside a `<p>`? In that case you could just divide the text in two, and have each paragraph as a `<p>`.

Something like:
```html
<p>
...
</p>
<hr />
<p>
...
</p> 
```